grammar.js

```diff
@@ -1580,7 +1582,21 @@ module.exports = grammar({
       repeat1($.guard),
     ),
 
-    comment: $ => token(seq('//', /.*/)),
+    comment: $ => seq(token('//'), choice(
+      $.using_directive,
+      $._comment_text,
+    )),
+
+    _comment_text: $ => token(prec(PREC.comment, /.*/)),
+
+    using_directive: $ => seq(
```

Comment:
Could you also add a test for it too here plz? - https://github.com/tree-sitter/tree-sitter-scala/blob/master/test/highlight/basics.scala

Reply:
Added to a separate file `comments.scala`